vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.kt

```diff
@@ -1072,13 +1082,17 @@ class VectorSettingsPreferencesFragment : PreferenceFragmentCompat(), SharedPref
                             }
 
                             override fun onMatrixError(e: MatrixError) {
-                                onDone(R.string.settings_fail_to_update_password)
+                                if (e.error == "Invalid password")
+                                    onDone(R.string.invalid_pass_error)
```

Comment:
It should be nice to have this error handled directly in the dialog. So the dialog would not be close automatically before the end of the request (you'll find how to do this here: https://stackoverflow.com/questions/2620444/how-to-prevent-a-dialog-from-closing-when-a-button-is-clicked)

Reply:
Ok Thank you . I would make the required changes